examples/java/README.md

```diff
@@ -0,0 +1,36 @@
+# Java Example
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/Kotlin/kotlinx-benchmark)
+
+## Project Structure
+
+Inside of this example, you'll see the following folders and files:
+
+```
+/
+├── build.gradle
+└── src/
+    └── main/
+        └── java/
+            └── test/
+                └── SampleJavaBenchmark.java
+```
+
+## Tasks
+
+All tasks can be run from the root of the project, from a terminal:
+
+| Task Name | Action |
+| --- | --- |
+| `gradle assembleBenchmarks` | Generate and build all benchmarks in the project |
```

Comment:
`gradle` command should be removed in the Task name column.
WDYT?

Reply:
I personally think it could be great to just copy and paste for folks using the terminal however I understand your point. ./gradlew does look a little bit messy however so I have omitted it as recommended.